scripts/docker-integration-tests/aggregator/test.sh

```diff
@@ -284,6 +303,46 @@ function test_aggregated_rollup_rule {
     retry_with_backoff prometheus_query_native
 }
 
+function test_metric_type_survives_aggregation {
+  now=$(date +"%s")
+
+  echo "Test metric type should be kept after aggregation"
+
+  # Emit values for endpoint /foo/bar (to ensure right values aggregated)
+  write_at="$now_truncated"
+  value="42"
+
+  metric_type="counter" \
+  prometheus_remote_write \
+  metric_type_test $now $value \
+  true "Expected request to succeed" \
+  200 "Expected request to return status code 200"
+
+  start=$(( $now - 3600 ))
+  end=$(( $now + 3600 ))
+  jq_path=".datapoints[0].annotation"
+
+  echo "Test query metric type"
+
+  # Test if metric type is stored in aggregated namespace
+  ATTEMPTS=50 TIMEOUT=2 MAX_TIMEOUT=4 \
+    namespace="agg" \
+    id='{__name__=\"metric_type_test\",label0=\"label0\",label1=\"label1\",label2=\"label2\"}' \
+    rangeStart=${start} \
+    rangeEnd=${end} \
+    jq_path="$jq_path" expected_value="CAEQAQ==" \
```

Comment:
Perhaps add a comment about the fact that this base64 value is the protobuf encoded value with the metric type on it?